src/app/projects/components/project-settings/components/embeddings/embeddings.component.ts

```diff
@@ -0,0 +1,155 @@
+import { Component, Input, OnInit, SimpleChanges } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { Project } from 'src/app/base/entities/project';
+import { ProjectApolloService } from 'src/app/base/services/project/project-apollo.service';
+import { Attribute } from '../../entities/attribute.type';
+import { DownloadedModel } from '../../entities/downloaded-model.type';
+import { Embedding } from '../../entities/embedding.type';
+import { DataHandlerHelper } from '../../helper/data-handler-helper';
+import { SettingModals } from '../../helper/modal-helper';
+
+@Component({
+  selector: 'kern-embeddings',
+  templateUrl: './embeddings.component.html',
+  styleUrls: ['./embeddings.component.scss'],
+})
+export class EmbeddingsComponent implements OnInit {
+
+  @Input() project: Project;
+  @Input() attributesArrayTextUsableUploaded: Attribute[];
```

Comment:
what exactly is this? 
an array of attributes that holds only useable && uploaded && text attributes?

Checking the code this more looks like a `useableTextAttributes` because useable is either useable, uploaded or automatically created.

Reply:
Yes, exactly that and it makes sense to rename it